src/main/scala/io/kf/etl/processors/common/converter/EntityConverter.scala

```diff
@@ -16,7 +16,10 @@ object EntityConverter {
       data_access_authority = study.data_access_authority,
       short_name = study.short_name,
       code = study.code,
-      domain = study.domain,
+      domain = study.domain match {
```

Comment:
you don't have to but you can do: `study.domain.fold(Seq.empty[String])(x => x.split(",").toSeq)`
the result is the same but some code coverage plugin will think you are not matching all cases and you are sure with fold that you return the same type in both sides of the equation

Reply:
yup